toolchain/check/merge.cpp

```diff
@@ -420,6 +439,17 @@ static auto CheckRedeclParamSyntax(Context& context,
       return false;
     }
   }
+  // The prefixes are the same, but the lengths may still be different.
+  // This only relevant for `impl` declarations where the final
+  // bracketing node is not included in the range of nodes being compared, and
+  // in those cases `diagnose` is false.
```

Comment:
The wrapping on this looks odd (particularly at "final bracketing", "bracketing" could be on the prior line), can you rewrap it please?

Reply:
Done.